examples/test-app/steps/DateOfMarriage.template.html

```diff
@@ -0,0 +1,26 @@
+{% extends "question-globals.njk" %}
+{% extends "look-and-feel/layouts/question.html" %}
+
+{% from "look-and-feel/components/fields.njk" import formSection, date %}
+
+{% set title %}
+{{ content.title }}
+{% endset %}
+
+{% block fields %}
+
+  {% call formSection() %}
+
+    {{ date(fields.date, content.date.question,
+      hideQuestion = true,
```

Comment:
Kind of confusing.

Reply:
All the fields have a label or a legend that describes them. You always want one as it helps with screen reader legibility but you don't always want to see it if the pages title describes it well enough:

```
What is your date of marriage? 

Date of Marriage
Day      Month     Year
[ 1   ]  [ 12   ]  [ 2017   ]
```

So I've been adding this `hideQuestion` option to visibly hide the label or legend as a way to "force" developers to actually give a label to describe the control but allow them to hide it.